### PR TITLE
Standardize info for collections maintained by the Ansible Cloud Team

### PR DESCRIPTION
**Pull request by alinabuzachis**
Standardize info for collections maintained by the Ansible Cloud Team

### CODE REVIEW

**Comment by jillr** [comment]
@alinabuzachis can you update this PR to include the team link like in 333?

**Comment by felixfontein** [comment]
(Marked as draft so we can figure out what to do - see https://forum.ansible.com/t/collection-maintainer-information-for-collections-included-in-ansible/2562)

File: 10/collection-meta.yaml [hunk 1]
```diff
@@ -28,9 +28,7 @@ collections:
     repository: https://github.com/ISIB-Group/inspur.sm
   kubernetes.core:
     maintainers:
-      - jillr
-      - akasurde
-      - gravesm
+      - ansible-collections/cloud
```
Comment (felixfontein) [inline 10/collection-meta.yaml:31]: Can you adjust this similar to https://github.com/ansible-community/ansible-build-data/pull/333/files ? Then we can finally merge this.

**Comment by felixfontein** [comment]
@alinabuzachis ping

**Comment by alinabuzachis** [comment]
> @alinabuzachis ping

@felixfontein Is there any update on being able to use a team instead of listing people?

File: 8/collection-meta.yaml [hunk 2]
```diff
@@ -125,15 +136,11 @@ collections:
...
-      - viciousprimate
-      - alinabuzachis
-      - GomathiselviS
-      - abikouo
-      - komaldesai13
+      maintainers:
+      - machacekondra
+      - mikemorency
+      - bardielle
+      - mariolenz
```
Comment (felixfontein) [inline 8/collection-meta.yaml:143]:
```suggestion
        - machacekondra
        - mikemorency
        - bardielle
        - mariolenz
```

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@alinabuzachis thanks a lot for your contribution!

**Pull request merged**